### DISCUSSION TITLE
Will this plugin work with a Smart Life enabled Water Shut off valve?

### DISCUSSION THREAD

**Discussion by saiajin** [milo526/homebridge-tuya-web#196] [Q&A] [answered]
Will this plugin work with a Smart Life enabled Water Shut off valve like this? 

Owfeel WIFI & Bluetooth Smart Water Valve, WIFI Control Water Valve Compatible With Alexa, Google and Application Program iOS/Android [https://www.amazon.com/dp/B08RS64NKV/ref=cm_sw_r_cp_api_glt_fabc_88NQJ2BTXQ870JGZ02H1?psc=1]()

**Comment by milo526** [comment, marked as answer]
The only way to know is to try or through the experience of others.

Please let us know if you find out!

**Reply by saiajin** [reply]
I was looking to purchase one that is compatible and was hoping someone else got a shut off valve to work with HK. Just trying to confirm compatibility b4 buying, but thanks.
